bin/runYANGgenericstats-daily.sh

```diff
@@ -12,6 +12,21 @@
 # License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 # either express or implied.
 
+wait_for_processes()
+{
+    PIDS=("$@")
+    max_processes=5
```

Comment:
make it three only. we have 4 cpus anyway and i d like to rather test it with smaller number

Reply:
done